### ISSUE TITLE
guessit crash

### ISSUE THREAD

**Issue by clashman** [guessit-io/guessit#427]
guessit.guessit('01.Seo Dong Yo_EP.01[].srt')

leads to:

```
===================== Guessit Exception Report =====================
version=2.1.1
string=01.Seo Dong Yo_EP.01[].srt
options={}
--------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/guessit/api.py", line 120, in guessit
    matches = self.rebulk.matches(string, options)
  File "/usr/local/lib/python3.6/site-packages/rebulk/rebulk.py", line 288, in matches
    self._execute_rules(matches, context)
  File "/usr/local/lib/python3.6/site-packages/rebulk/rebulk.py", line 319, in _execute_rules
    rules.execute_all_rules(matches, context)
  File "/usr/local/lib/python3.6/site-packages/rebulk/rules.py", line 318, in execute_all_rules
    when_response = execute_rule(rule, matches, context)
  File "/usr/local/lib/python3.6/site-packages/rebulk/rules.py", line 343, in execute_rule
    rule.then(matches, when_response, context)
  File "/usr/local/lib/python3.6/site-packages/guessit/rules/processors.py", line 222, in then
    if match.raw[0] in seps_no_groups and (len(match.raw) < 3 or match.raw[2] not in seps_no_groups):
IndexError: string index out of range
```

**Comment by ratoaq2** [comment]
This was already fixed in guessit 2.1.2

**Comment by ratoaq2** [comment]
Duplicates #385
